tests/Multiple/ConnectionManagerConcurrentTest.php

```diff
@@ -16,6 +16,10 @@ public function testEmptyListsThrows()
 
     public function testWillForwardToInnerConnector()
     {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('Not supported on HHVM with Promise v3');
+        }
```

Comment:
Looks like a valid option and HHVM definitely doesn't deserve any priority for this project. As an alternative, perhaps we should install Promise v2 for HHVM in our test setup instead? See https://github.com/reactphp/promise-timer/pull/67, https://github.com/reactphp/reactphp/pull/524 and others.